client-ts/package.json

```diff
@@ -35,13 +35,13 @@
     "gulp-babel": "^7.0.0",
     "gulp-rename": "^1.2.2",
     "gulp-typescript": "^3.1.3",
-    "jasmine": "^2.5.2",
+    "jasmine": "^2.8.0",
```

Comment:
@Eilon - I updated versions of the msgpack5 and jasmine modules. The first contains a fix for bug #945. The second is needed because it has support for verifying typed arrays.

Reply:
Ok so just these 2 got updated? (jasmine + msgpack5)

Reply:
Yes - they were updated to newest versions